plugins/modules/pnp_intent.py

```diff
@@ -352,15 +359,17 @@ def get_site_type(self):
                 params={"name": self.want.get("site_name")},
             )
         except Exception:
+            self.log("Site {0} not found".format(self.want.get("site_name")), "CRITICAL")
```

Comment:
self.log("Exception occurred as site '{0}' was not found".format(self.want.get("site_name")), "CRITICAL")

Reply:
Resolved